go/pkg/rexec/rexec_test.go

```diff
@@ -441,71 +441,6 @@ func TestGetOutputFileDigests(t *testing.T) {
 	}
 }
 
-func TestGetOutputDigests(t *testing.T) {
```

Comment:
Why remove this test?

Reply:
I can add a test if you think it makes sense. I was just thinking that there's not really any new logic here since I changed it so that this function no longer computes the directory digest. Instead it is just exposing whatever is in the ActionResult after a remote execution has been run.

Reply:
Yeah I think we should try to always have tests for our public functions to maintain coverage.

Reply:
I added the test, but currently the fake server doesn't support output directories so I left a TODO on line 474.

Reply:
Test no longer needed since the function is removed.